Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -522,9 +521,9 @@ void ChessWidget::playback_move(PlaybackDirection direction)
     update();
 }
 
-DeprecatedString ChessWidget::get_fen() const
+String ChessWidget::get_fen() const
```

Comment:
I'm not a fan of having a function returns a raw `String`, it's not better than a `DeprecatedString` IMO as someone will still need to make this a `ErrorOr<String>` later on. I would rather put the `release_value_but_fixme_should_propagate_errors()` on the caller side